generator/src/main.rs

```diff
@@ -2171,12 +2181,17 @@ fn xr_var_ty(member: &Member) -> TokenStream {
         quote! { #ident }
     };
     let mut ty = if let Some(ref len) = member.static_array_len {
+        // If the len is a constant, we see if it is an aliased constant. Since we do not emit
+        // aliased constant declarations, we must replace the alias with the original constant name
+        // that we do emit.
+        let len = match api_aliases.iter().find(|(alias, _)| alias == len) {
+            Some((_, og)) => og,
+            _ => len,
+        };
         if let Ok(len) = len.parse::<usize>() {
             quote! { [#ty; #len] }
         } else {
-            assert!(
-                len.starts_with("XR_MAX_") || len.ends_with("_COUNT") || len.ends_with("_SIZE_FB")
-            );
+            assert!(len.starts_with("XR_"));
```

Comment:
@Ralith  The only invariant needed here is that the constant name starts with `XR_` so we can remove it with `len[3..]` below. I changed this so I didn't need to add the a check for `len.ends_with("_COUNT_FB") || len.starts_with("XR_UUID")`.